core/src/main/java/org/apache/brooklyn/location/ssh/SshMachineLocation.java

```diff
@@ -152,6 +152,12 @@
     public static final ConfigKey<Boolean> DETECT_MACHINE_DETAILS = ConfigKeys.newBooleanConfigKey("detectMachineDetails",
             "Attempt to detect machine details automatically. Works with SSH-accessible Linux instances.", true);
 
+    public static final ConfigKey<Boolean> BYON_USER_RESET_LOGIN = ConfigKeys.newBooleanConfigKey(
```

Comment:
this should be a config on `CreateUserPolicy` as that is the only thing it affects, and it affects everything there

on a different note if that policy is tied to jclouds it will have no effect on BYON deployments.  if it isn't related to jclouds it should probably be moved to the policies project and a different package.  (we should leave the old instance in place as an empty class extending the item in the new location, but deprecated, so that references to the old one will still work.)